Assets/Scripts/Environment/PlayerSafeZone.cs

```diff
@@ -1,12 +1,22 @@
 ﻿using UnityEngine;
 using System.Collections;
 
+/// <summary>
+/// The player safe zone class ???
```

Comment:
@IAmAllOfYou Can you write summaries as comments for this class and the functions